build.gradle

```diff
@@ -225,6 +228,7 @@ project('client') {
         exclude "**/impl/**";
         options.addBooleanOption('html5', true)
         options.addBooleanOption("Xdoclint:all,-reference", true)
+        apply plugin: 'io.franzbecker.gradle-lombok'
```

Comment:
I didn't realize this was possible! Can we substitute all of the 'dependencies' sections under 'javadoc' that point to lombok with this? If so that would be a major improvement.

Reply:
If we add the plugin to the javadoc sections, do we need to declare it here:

https://github.com/pravega/pravega/blob/cbb90501a350db0ffe3404abeb8557c7a2626587/build.gradle#L48

Reply:
Actually tried to get build working with just the javadoc sections, but it didn't work. Removing the plugin from the javadoc sections didn't change anything either. So the javadoc entries were actually extraneous.  So I removed them. Maybe when we get to Java 10 it will work.